ingestion/src/metadata/test_suite/validations/table/pandas/tableCustomSQLQuery.py

```diff
@@ -28,4 +28,4 @@ class TableCustomSQLQueryValidator(
 
     def _run_results(self, sql_expression: str):
         """compute result of the test case"""
-        return self.runner.query(sql_expression)
+        return self.runner[0].query(sql_expression)
```

Comment:
You should run the `sql_expression` against every chunk and aggregate the rows return by the expression and return those rows.

Reply:
yep, fixed it..

I was running tests on parquet, it brought me whole list of dataframe
recently found out that parquet is not being sent as chunks, will fix it in another PR